src/app/app-designs/app-designs.component.scss

```diff
@@ -27,20 +27,43 @@
 
 .table {
   width: 100%;
+  .mat-cell:first-child,
+  .mat-header-cell:first-child {
```

Comment:
I think is better if you create a class, because with this you are modifying all the tables.